### PR TITLE
f4pga/wrappers/sh/quicklogic: cleanup

### PR DESCRIPTION
**Pull request by umarcor**

### CODE REVIEW

File: f4pga/wrappers/sh/quicklogic/env [hunk 1]
```diff
@@ -16,10 +16,7 @@
...
 # SPDX-License-Identifier: Apache-2.0
 
-MYPATH=`realpath $0`
-MYPATH=`dirname ${MYPATH}`
-
 if [ -z $VPRPATH ]; then
-	export VPRPATH=$MYPATH
-	export PYTHONPATH=${VPRPATH}/python:${VPRPATH}/python/prjxray:${PYTHONPATH}
+  export VPRPATH="$F4PGA_BIN_DIR"
+  export PYTHONPATH="${VPRPATH}/python:${VPRPATH}/python/prjxray:${PYTHONPATH}"
```
Comment (lpawelcz) [inline f4pga/wrappers/sh/quicklogic/env:21]: I think we can remove `${VPRPATH}/python/prjxray` from `PYTHONPATH`

**Review by lpawelcz — CHANGES_REQUESTED** [review]
Looks good to me except `prjxray` in `PYTHONPATH`

**Review by lpawelcz — APPROVED** [review]
LGTM!

File: f4pga/wrappers/sh/quicklogic/env (hunk 1 above)
Comment (umarcor) [reply f4pga/wrappers/sh/quicklogic/env]: Done!

**Comment by umarcor** [comment]
Signed-off-by: Unai Martinez-Corral <umartinezcorral@antmicro.com>

**Pull request merged**